tests/pipeline/test_AE.py

```diff
@@ -35,5 +35,21 @@ def test_results(self, demo_dir):
                 """.format(output_dir, output_dir)
         r = runR(r_cmd, demo_dir)
         assert "class: OutriderDataSet" in r.stdout
-        assert "dim: 441 12" in r.stdout
-        assert "res: 5292 15" in r.stdout
\ No newline at end of file
+        assert "dim: 431 10" in r.stdout
+        assert "res: 4310 15" in r.stdout
```

Comment:
why do the results change so much?

Reply:
We are working with different subsets. Before the group `outrider` had 12 samples, 2 imported, now it has 10